tools/cockpit.spec

```diff
@@ -424,6 +427,14 @@ cluster. Installed on the Kubernetes master. This package is not yet complete.
 
 %endif
 
+%package ostree
+Summary: Cockpit user interface for rpm-ostree
```

Comment:
I held off on doing this since the version we would need to depend on doesn't exist yet. Also adding the dependency will pull rpm-ostree into all our test systems, we probably need a skip mechanism on the other OSes.

On the other hand this packages is really only for atomic and you can't have an atomic system without rpm-ostree. Installing this packages without rpm-ostree just results in the getting the curtain message.